### PR TITLE
fix kubeedge license

### PR DESCRIPTION
**Pull request by zc2638**
**What type of PR is this?**

/kind bug

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: cloud/pkg/edgecontroller/controller/upstream.go [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2014 The Kubernetes Authors.
+Copyright 2020 The Kubeedge Authors.
```
Comment (subpathdev) [inline cloud/pkg/edgecontroller/controller/upstream.go:2]: can you use the current year with 2021?

Comment (zc2638) [reply cloud/pkg/edgecontroller/controller/upstream.go:2]: My fault, it should be 2019.This file was created at that time.

Comment (subpathdev) [reply cloud/pkg/edgecontroller/controller/upstream.go]: Why we have to use the date, when the file was created? Can we used the last modification date instead? And that is in the current year ....

Comment (zc2638) [reply cloud/pkg/edgecontroller/controller/upstream.go]: `Copyright` usually starts with the first date and ends with the second date.

Comment (subpathdev) [reply cloud/pkg/edgecontroller/controller/upstream.go]: Yes you are right

**Review by fisherxu — COMMENTED** [review]
This is because we copied some codes like `updatePodStatus` from K8s/kubelet, so we used the k8s license here. So can we add a new line for kubeedge and keep the k8s here?

**Comment by zc2638** [comment]
> This is because we copied some codes like `updatePodStatus` from K8s/kubelet, so we used the k8s license here. So can we add a new line for kubeedge and keep the k8s here?

I suggest no need to keep.
Like other projects that quote or directly copy part of the k8s code, most of them use their own license.
I did not find anything about kubernetes in the license.
What do you think?

**Comment by fisherxu** [comment]
> > This is because we copied some codes like `updatePodStatus` from K8s/kubelet, so we used the k8s license here. So can we add a new line for kubeedge and keep the k8s here?
> 
> I suggest no need to keep.
> Like other projects that quote or directly copy part of the k8s code, most of them use their own license.
> I did not find anything about kubernetes in the license.
> What do you think?

In fact in edged we also keep to use the K8s license in most files, if you copy the code from a project, it's necessary to make declaration.

**Comment by kevin-wangzefeng** [comment]
Respecting the copyright of upstream code is required according to Apache 2.0 license, we shall not remove tuat.

Adding a new line to claim KubeEdge copyright shall be fine.

**Comment by zc2638** [comment]
> Respecting the copyright of upstream code is required according to Apache 2.0 license, we shall not remove that.
> 
> Adding a new line to claim KubeEdge copyright shall be fine.

Okey.

File: cloud/pkg/edgecontroller/controller/upstream.go [hunk 2]
```diff
@@ -1,4 +1,5 @@
 /*
+Copyright 2019 The Kubeedge Authors.
```
Comment (fisherxu) [inline cloud/pkg/edgecontroller/controller/upstream.go:2]:
```suggestion
Copyright 2019 The KubeEdge Authors.
```

Comment (zc2638) [reply cloud/pkg/edgecontroller/controller/upstream.go]: done.

**Review by fisherxu — COMMENTED** [review]
Need rebase to master to fix the failed tests :)

**Review by fisherxu — COMMENTED** [review]
/approve

**Review by fisherxu — COMMENTED** [review]
/lgtm

**Pull request merged**